R/InstanceSettings.R

```diff
@@ -37,13 +37,13 @@ InstanceSettings <- R6::R6Class( # nolint object_name_linter
         "db_user_password", # api
         "lamindb_version" # api
       )
-      missing_column <- setdiff(expected_keys, names(settings))
-      if (length(missing_column) > 0) {
-        cli_abort("Missing column: ", missing_column)
+      missing_key <- setdiff(expected_keys, names(settings))
+      if (length(missing_key) > 0) {
+        cli_abort(paste0("Missing key: ", paste(missing_key, collapse = ", ")))
```

Comment:
Missing key**s**? I think **{cli}** can automatically pluralise stuff if you want to get fancy but not sure it's worth the effort.

Reply:
Oh, this is cool, I didn't know `{cli}` could do this!